### PR TITLE
T17076

### PR DESCRIPTION
**Pull request by tchx84**
https://phabricator.endlessm.com/T17076

### CODE REVIEW

File: data/css/blog.scss [hunk 1]
```diff
@@ -122,8 +122,9 @@ $highlighted-background-color: darken($background-dark-color, 5%) !default;
 
 .article-page {
   .related-posts {
-    margin: 0px 50px 20px 50px;
-
+    background-color: $background-light-color;
+    margin-right: 15px;
+    padding: 0px 50px 20px 50px;
```
Comment (ptomato) [inline data/css/blog.scss]: Why the change from `margin` to `padding`? It would be good to give a rationale for a non-obvious change like that in the commit message.

Comment (tchx84) [inline data/css/blog.scss]: @ptomato let me try to sketch it here, hehe:

1) this change is about adding a background color to the Related Articles content goup.
2) previously, we had that margin applied to content group, so when adding a background color, the background would only extend to immediate boundaries of the content goup (looking like a floating box on top of the article background color).
3) so, to fix that I switch to padding so we would have the spacing around the content but extending its background on top of the articles background.
4) lastly, the final margin-right is simply to prevent the content group to hide the scrolled window slider.

does this make sense?

Comment (ptomato) [inline data/css/blog.scss]: OK, got it, could you please add it to the commit message then?

**Comment by ptomato** [comment]
Per @katelk28's comment on https://phabricator.endlessm.com/T17076, it might be good to add a style class to the default theme for content groups like this one that pop up over the webview.

**Comment by xjuan** [comment]
margin -> padding is because it needs to cover the article with the background color

CR pass
